index.bs

```diff
@@ -409,7 +410,7 @@ See <a href="https://lists.w3.org/Archives/Public/public-speech-api/2012Sep/0072
     1. Let <var>promise</var> be <a>a new promise</a>.
     1. Initiate the download of the on-device speech recognition language for <var>lang</var>.
         <p class=note>
-          Note: The user agent can prompt the user for explicit permission to download the on-device speech recognition language pack.
+          Note: The user agent MAY prompt the user for explicit permission to download the on-device speech recognition language pack.
```

Comment:
You're not supposed to use MAY inside a non-normative paragraph like a note, and in fact you should turn on [Complain About: accidental-2119](https://speced.github.io/bikeshed/#metadata-complain-about) in the [bikeshed metadata](https://speced.github.io/bikeshed/#metadata) to flag when you do it by accident. Instead, you'll need to specify the actions the UA must take if the user denies permission, similar to "If the user explicitly or implicitly signals that they do not want to start the download" in https://pr-preview.s3.amazonaws.com/webmachinelearning/writing-assistance-apis/pull/47.html#step-download-user-interface.

Reply:
I had it enabled in another PR but forgot to merge, this is now done (with another lint).